OSDOCS9493: Added configuring certificates to secure routes in your cluster in release notes (#78572)
openshift/openshift-docs: security/cert_manager_operator/cert-manager-operator-release-notes.adoc

With this release, FIPS mode is now automatically enabled for {cert-manager-operator}. When installed on an {product-title} cluster in FIPS mode, {cert-manager-operator} ensures compatibility without affecting the cluster's FIPS support status.

*Securing routes with cert-manager managed certificates (Technology Preview)*

> With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].

Reviewer: ```suggestion
With this release, you can manage certificates referenced in `Route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].
```

The NCM issuer is validated only with version 1.1.1 and the {cert-manager-operator} version 1.14.0. This version handles tasks such as issuance, renewal, and managing certificates for the API server and ingress controller of {product-title} clusters.
====

> *Configuring certificates to secure routes in your cluster(Technology Preview)*

Reviewer: ```suggestion
*Configuring certificates to secure routes in your cluster (Technology Preview)*
```

> *Configuring certificates to secure routes in your cluster (Technology Preview)*

Reviewer 2: ```suggestion
*Securing routes with cert-manager managed certificates (Technology Preview)*
```

> With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the cert-manager Operator for Red Hat OpenShift].

Reviewer 2: How about
```suggestion
With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].
```

Author: Might have some rendering issues. Will check.

> With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The OpenShift router promptly serves renewed certificates. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].

Reviewer: ```suggestion
With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```

> With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].

Reviewer: ```suggestion
With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors, and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```
